db/schema.rb

```diff
@@ -10,14 +10,17 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 5) do
+ActiveRecord::Schema.define(version: 4) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "mission_tasks", force: :cascade do |t|
     t.bigint "mission_id"
     t.bigint "task_id"
+    t.string "message"
+    t.string "image_path"
+    t.boolean "completed?"
```

Comment:
took these from the tasks table so when the user interacts completes a task it will update the mission task and not the task which can have many missions